plugin/trino-bigquery/README.md

```diff
@@ -22,9 +22,16 @@ You can follow the steps below to be able to run the integration tests locally.
   **BigQuery Admin** role assigned.
 * Get the base64 encoded text of the service account credentials file using `base64
   /path/to/service_account_credentials.json`.
+* Create a new BigQuery `ClOUD_RESOURCE` connection and grant the connection service account GCS permissions.
+  [Documentation](https://cloud.google.com/bigquery/docs/create-cloud-resource-connection).
+   * `bq mk --connection --location=us --project_id=$PROJECT_ID --connection_type=CLOUD_RESOURCE $CONNECTION_ID`
+   * Now we need to grant the new connection's service account the GCS permissions.
+   * To do this, run `bq show --connection $PROJECT_ID.us.$CONNECTION_ID`, which will display the service account ID.
+   * Grant the service account GCS Object Viewer permissions. e.g.: `gsutil iam ch serviceAccount:bqcx-xxxx@gcp-sa-bigquery-condel.iam.gserviceaccount.com:objectViewer gs://DESTINATION_BUCKET_NAME`
```

Comment:
I assume `bqcx-xxxx@gcp-sa-bigquery-condel` is actually `$CONNECTION_ID@$PROJECT_ID`? If so please change it to be more generic otherwise it's hard to figure out what values are literals vs which need to be replaced.

Reply:
No. It's the output of the `bq show` command, which displays the underlying service account of the connection, on which the permissions need to be granted.  I also clarified how to set the variable while running the tests.